src/views/QuizView/quizView.js

```diff
@@ -93,9 +99,22 @@ function renderQuizData() {
   for (let i = 0; i < 4; i++) {
     answersContainer.appendChild(Button(answers[i], 'answer', false, 'click', nextQuestion));
   }
+  date = new Date();
+  startTime = date.getTime();
 }
 
-function nextQuestion() {
+function saveAnswer(answer) {
+  date = new Date();
+  endTime = date.getTime();
```

Comment:
this part creates 2 separate Date objects - it may be a good idea to have just one of them initialized with the Timer component. For our use case, we can use the timer to get timestamp